pkg/types/cciptypes/models.go

```diff
@@ -0,0 +1,38 @@
+package cciptypes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type Address string
+
+func (a Address) Equals(addr2 Address) bool {
+	if common.IsHexAddress(string(a)) && common.IsHexAddress(string(addr2)) {
+		return common.HexToAddress(string(a)) == common.HexToAddress(string(addr2))
+	}
+	return a == addr2
+}
+
+func (a Address) ToEVM() (common.Address, error) {
```

Comment:
this needs to be a func in the evm relayer, not a method on the type in this repo